### ISSUE TITLE
"determinism" in Chapter 1

### ISSUE THREAD

**Issue by owoodman** [OSTP/PhysicsArtofModelling#158]
I think we should clarify what determinism means in this paragraph. Maybe just rephrase it to "This could only be done at the expense of determinism, the idea that we can predict how particular situations evolve in time. This led to a theory that could only predict the _probabilities_ that certain outcomes will be realized."  

Quantum mechanics is a theory that was developed in the 1920s to incorporate Einstein's conclusion that light is made of particles (or rather, quantized lumps of energy called quanta) and describe Nature at the smallest scales. **This could only be done at the expense of determinism, leading to a theory that could not predict how particular situations evolve in time, but only the probabilities that certain outcomes will be realized**. Quantum mechanics was further refined during the twentieth century into Quantum Field Theory, which led to the Standard Model of particle physics that describes our current understanding of matter through the theories of the electroweak and strong forces.

**Comment by ryanmartinneutrino** [comment]
yup, that's fine with me if you want to make that change.
